.github/workflows/zips.yml

```diff
@@ -223,6 +218,31 @@ jobs:
           edit-mode: replace
           body: Build files for ${{ github.event.pull_request.head.sha }} have been deleted.
 
+  dryrun-publish-to-wporg:
+    name: '[DRY RUN] Publish to WordPress.org'
+    runs-on: ubuntu-latest
+    # environment: do not define to omit SVN credentials + deployment protection rules for this dry-run.
+    if: github.event_name == 'workflow_dispatch' && ( github.ref_type == 'tag' || inputs.release_version != '' )
+    needs: build-zips
+    timeout-minutes: 20
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: zip-files
+          path: /tmp
+      - name: Extract
+        run: |
+          unzip /tmp/google-site-kit.zip
+      - uses: 10up/action-wordpress-plugin-deploy@2.2.2
+        with:
+          dry-run: true
+        env:
+          BUILD_DIR: ./google-site-kit
+          SLUG: google-site-kit
+          SVN_PASSWORD: no-op-password
+          SVN_USERNAME: no-op-username
+          VERSION: ${{ inputs.release_version }}
+  
   publish-to-wporg:
```

Comment:
This job currently has the same `needs` dependency of `build-zips` (same as dry-run meaning they'd run in parallel). It could use the dry-run job as a dependency instead but I'm not sure it's necessary.